spec.md

```diff
@@ -151,6 +151,17 @@ Service Broker MAY reject the request with `412 Precondition Failed` and
 provide a message that informs the operator of the API version that is to be
 used instead.
 
+## Extending Resources
+
+Senders of messages defined by this specification MAY include additional
```

Comment:
We should have a first-class place to put extension data on requests/responses, otherwise it will be very cumbersome to implement clients that respect extensions.

Reply:
I disagree with this. I don't think you need to be that prescriptive.

Reply:
I have had a change of heart. I am flipping in support of the more prescriptive idea that extension fields MUST be placed inside a extension wrapper (aka "a first-class place to put extension data on requests/responses"). 

With the caveat that I do not want to prevent a broker from adding additional fields in the json payload. After careful thought, I think those additional fields must not be applicable to the extensions mechanism.